### PR TITLE
bugfix 2810 – duplicate title downloadable pdp

### PR DESCRIPTION
**Pull request by winniepukki**
Fixes scandipwa/scandipwa#2810

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
Now placeholder is displayed
Screenshot 2021-07-01 at 14 44 24

**Comment by winniepukki** [comment]
@aleksandrakorolova the behaviour is too random. Unable to reproduce

**Comment by winniepukki** [comment]
Please leave a product link you are testing and in how many cases of 10 you succeed reproducing?

File: packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ export class ProductAttributeValue extends PureComponent {
 
     renderBooleanAttribute() {
         const { attribute: { attribute_value } } = this.props;
-        return this.renderStringValue(attribute_value ? __('Yes') : __('No'));
+        return this.renderStringValue(+attribute_value ? __('Yes') : __('No'));
```
Comment (ejnshtein) [inline packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js:104]: If you want to convert a value from string to number, better to use `parseInt` of `parseFloat` functions, using `+` in from of a variable is an implicit way of doing this.

**Review by ejnshtein — CHANGES_REQUESTED** [review]
Overall looks OK, just need an answer to one comment

**Comment by carinadues** [comment]
Closed because the issue isn't reproducible anymore.

**Comment by carinadues** [comment]
The issue was fixed in #2977

**Pull request closed without merging**